src/m365/spo/commands/file/FileSharingLinkUtil.ts

```diff
@@ -0,0 +1,33 @@
+import request, { CliRequestOptions } from "../../../../request";
+import { formatting } from "../../../../utils/formatting";
+import { urlUtil } from "../../../../utils/urlUtil";
+import { GraphFileDetails } from "./GraphFileDetails";
+
+export class FileSharingLinkUtil {
```

Comment:
I'm not sure if this is the right place to put this, can you assist me @pnp/cli-for-microsoft-365-maintainers?

Context:
We have few commands spo file sharinglink `get`, `set`, `add`, `list`, `clear`, `remove`. They all have to get the Graph ID for the list and file where this file is stored. It's always the same request to retrieve these IDs. It would make sense to centralize it, but should we put it in the `spo file` folder?

Reply:
I suggest we add the function to the existing `spo` util.